.cirrus.yml

```diff
@@ -19,7 +19,14 @@ task:
   codecov_script:
     - curl -Os https://cli.codecov.io/latest/linux/codecov
     - chmod +x codecov
+  rdma_core_script:
+    - dnf install -y git libnl3-devel libudev-devel make pkgconfig valgrind-devel
+    - git clone https://github.com/RDMA-Rust/rdma-core.git -b fix-rxe-next-poll
```

Comment:
Should we update the URL of rdma-core to the upstream one since this commit has been merged?

Reply:
Done